### ISSUE TITLE
Make server IP/Port configurable

### ISSUE THREAD

**Issue by dbrgn** [coredump-ch/rpsrtsrs#51]
Put it in the menu, and maybe persist it in a configfile.

**Comment by rnestler** [comment]
Yes we definitely need some way to have text input in the menu! I was thinking about using [conrod](https://github.com/PistonDevelopers/conrod) for that.
